examples/crm/src/sales/SalesList.tsx

```diff
@@ -0,0 +1,66 @@
+import Stack from '@mui/material/Stack';
+import {
+    BooleanField,
+    CreateButton,
+    DatagridConfigurable,
+    DeleteWithConfirmButton,
+    EditButton,
+    ExportButton,
+    List,
+    SearchInput,
+    TextField,
+    TopToolbar,
+    useGetIdentity,
+    useRecordContext,
+} from 'react-admin';
+import { Sale } from '../types';
+import { TransferAdminRole } from './TransferAdminRole';
+import { useIsAdmin } from './useIsAdmin';
+
+function SalesListActions() {
+    return (
+        <TopToolbar>
+            <ExportButton />
+            <CreateButton variant="contained" label="New Sale" />
+        </TopToolbar>
+    );
+}
+
+const filters = [<SearchInput source="q" alwaysOn />];
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏I don't know if it's possible but I can't search via prenon + name. Could we do better or are we limited by fakerest?